draft-ietf-tls-ticketrequests.md

```diff
@@ -157,7 +157,7 @@ negotiate a fresh session (full handshake).
 
 resumption_count
 : The number of tickets desired by the client when the server is willing to
-resume using the presented ticket.
+resume using a ticket presented in this ClientHello.
```

Comment:
My initial change here was to add a parenthetical "if any", since there may not be any presented tickets.  On the other hand, a quick read of 8446 didn't find any limitation on "at most one ticket in a given 'pre_shared_key' extension", so in principle there is not a single specific "presented ticket", and the "if any" parenthetical seems to not flow very well anymore after this change.